server/helpers/loadFrontendConfig.js

```diff
@@ -20,4 +20,5 @@ module.exports = {
   ADALITE_ENABLE_DEBUGGING: process.env.ADALITE_ENABLE_DEBUGGING === 'true',
   ADALITE_ENABLE_TREZOR: !!process.env.ADALITE_ENABLE_TREZOR,
   ADALITE_APP_VERSION: appVersion,
+  ADALITE_LOGOUT_AFTER: process.env.ADALITE_LOGOUT_AFTER * 1000,
```

Comment:
I would rather multiply it by 1000 directly in setTimeout. Modifying the value there seems a bit confusing to me. I would use parseInt for the sake of consistency and check ADALITE_LOGOUT_AFTER for falsiness rather than for >0 in app.js